ui5/browser/controller/Browser.controller.js

```diff
@@ -505,8 +507,8 @@ sap.ui.define(['sap/ui/core/Component',
 
           this.websocket.Send('CHDIR:' + path);
 
-          this.chdir = true;
```

Comment:
@linev Here you are deleting my global var, that was introduce to debug the fact that when you double click on a folder and so switch to it, you can still unfold the subfolders.
By removing that i am afraid that this is not working anymore, is that something that you tried ?

Reply:
You could try to checkout my branch and test it. In my mind - everything works now

Reply:
@linev I just checkout on your branch and you mind is true everything works

Reply:
If you remove my var this bit of code is useless, and you can remove it : 
```
if(this.chdir) {
  this.chdir = false;
  this.model.clearFullModel();
  this.model.reloadMainModel(true);
}
```
lines 657-662

Reply:
@Falcort 
You could check why icons for ROOT objects in TFile now shown.
Also you could try to apply vertical layout to items in master page.
Now TreeTable always expands over available vertical space

Reply:
For the TreeTable expanding, it is a SAP UI 5 bugs, that i am actually resolving
I will check the icons after solving the TreeTable size issue

Reply:
@Falcort Problem with icons is resolved. I will push next changes soon